issue_templates/config.yml

```
@@ -2,7 +2,7 @@ blank_issues_enabled: true
 contact_links:
   - name: Discord
     about: Ask questions on our Discord Server.
-    url: https://discord.gg/modrinth-734077874708938864
+    url: https://discord.gg/modrinth
```

Could this actually be changed to `https://discord.modrinth.com`? In case we lose the vanity URL, it's much easier to change a single Cloudflare setting than to update the link everywhere.

```suggestion
    url: https://discord.modrinth.com
```

---

Done ^^